app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PPOCardView.kt

```diff
@@ -415,7 +415,7 @@ fun ConfirmAddressButtonsView(isConfirmButtonEnabled: Boolean, onEditAddressClic
             modifier = Modifier
                 .weight(0.5f),
             onClickAction = { onEditAddressClicked.invoke() },
-            text = stringResource(id = R.string.edit_fpo),
+            text = stringResource(id = R.string.discovery_favorite_categories_buttons_edit),
```

Comment:
The french string for "Edit" was translated wrong. While the translation folx corrected the translation, for some reason it won't update on our end, so i found this already translated string for "Edit" that is correct. we can use this for now until the other translation stuff is sorted.